src/main/java/org/spongepowered/api/command/Command.java

```diff
@@ -63,10 +63,10 @@
  * as required. This <em>may</em> enable the use of client side command
  * completions, if the implementation is equipped to do so.</p>
  *
- * <p>Commands in Sponge are provided with a {@link Cause}, which explains
- * who <strong>directly</strong> invoked the command. In line with causes used
- * in events, you may assume that the {@link Cause#root()} is the
- * <strong>intended</strong> direct invoker.</p>
+ * <p>Commands in Sponge are provided with a {@link CommandCause}, which
```

Comment:
May need re-wording.

Reply:
Opted to move this part of the description to the CommandCause interface, makes sense to lump that all together.